### ISSUE TITLE
Squiz.WhiteSpace.OperatorSpacing does not allow newlines before `?`

### ISSUE THREAD

**Issue by oliverklee** [squizlabs/PHP_CodeSniffer#3377]
**Describe the bug**
The rule `Squiz.WhiteSpace.OperatorSpacing` does not allow splitting the ternary conditional operator over multiple lines (which is needed if the line is getting too long for PSR-12).

**Code sample**
```php
$ruleType = (!$cssRule->hasNonEmptyMedaiQuery()) && !$hasUmatchablePseudo)
    ? 'inlinable' : 'uninlinable';
```

**Custom ruleset**
```xml
<?xml version="1.0"?>
<ruleset name="My Custom Standard">
  <rule ref="Squiz.WhiteSpace.OperatorSpacing"/>
</ruleset>
```

**To reproduce**
1. Create a file called `test.php` with the code sample above.
2. Run `phpcs test.php`
3. See error message displayed
```
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 4 | ERROR | [x] Expected 1 space before "?"; newline found
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

**Expected behavior**
No warning.

**Versions (please complete the following information):**
 - OS: Ubuntu
 - PHP: 7.4.20
 - PHPCS: 3.5.8
 - Standard: Squiz

**Additional comments**
Great issue template! :heart:

**Comment by jrfnl** [comment]
Have you tried using the `ignoreNewlines` option ?
See: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Customisable-Sniff-Properties#squizwhitespaceoperatorspacing

**Comment by oliverklee** [comment]
Oh, this indeed does the trick. Totally missed this. Thanks for the quick and helpful reply! :heart:
